src/ngraph/frontend/onnx_import/utils/convpool.cpp

```diff
@@ -15,10 +15,8 @@
 //*****************************************************************************
 
 #include <cmath>
-#include <string>
 #include <unordered_map>
 
-#include "convpool.hpp"
 #include "core/attribute.hpp"
 #include "core/node.hpp"
```

Comment:
Both `attribute.hpp` and `node.hpp` are duplicated - they're already in header file.

Similarly `shape.hpp`, `coordinate_diff.hpp`